lib/bibliotheca/exercise.rb

```diff
@@ -8,6 +13,19 @@ def effective_language_name(guide)
       language || guide.language.name
     end
 
+    def process_choices!
+      multiple_choices_to_test! if multiple_choice?
+    end
+
+    def multiple_choices_to_test!
+      value = choices.select { |choice| choice['checked'] }.map { |choice| choice['value']}.join(':')
+      self.test = { 'equal' => value }.to_yaml
```

Comment:
The problem with this approach - joining the choices values - is that is bugs prone.

For example, lets's suppose that the options are: 

* La expresión es incorrecta: no tipa
* La expresión es incorrecta: no compila
* Las x's de expresión son zaraza
* ¿La respuesta es "Japón"?   

Most of them will have problems when interpolating: 

1. The first and second contain colons, which will break the interpolation
2. All of them contain no ascii characters, which may lead to encoding problems
3. The third and fourth options contain single and double quotes, which may break the yaml parser.  

For those reasons, Atheneum is no sending the joined values, but the joined indexes: https://github.com/mumuki/mumuki-atheneum/blob/master/app/assets/javascripts/application/multiple-choice.js

I recommend following that strategy here too, or changing it in atheneum.